### ISSUE TITLE
Cross compiler broken with recent GCC update

### ISSUE THREAD

**Issue by mscastanho** [advancetoolchain/advance-toolchain#2017]
The build of gcc_1 started to fail on cross targets with recent GCC updates (see #1977 and #1990):
```
In file included from /home/tcbot/bot-worker/at/src/at15.0-0-alpha.ubuntu-20_x86_64_ppc64le/sources/gcc/libgcc/config/rs6000/_kf_to_sd.c:37:
/home/tcbot/bot-worker/at/src/at15.0-0-alpha.ubuntu-20_x86_64_ppc64le/sources/gcc/libgcc/dfp-bit.c:32:10: fatal error: stdio.h: No such file or directory
   32 | #include <stdio.h>
      |          ^~~~~~~~~
compilation terminated.
make[3]: *** [/home/tcbot/bot-worker/at/src/at15.0-0-alpha.ubuntu-20_x86_64_ppc64le/sources/gcc/libgcc/static-object.mk:17: _kf_to_sd.o] Error 1
make[3]: *** Waiting for unfinished jobs....
In file included from /home/tcbot/bot-worker/at/src/at15.0-0-alpha.ubuntu-20_x86_64_ppc64le/sources/gcc/libgcc/config/rs6000/_kf_to_dd.c:37:
/home/tcbot/bot-worker/at/src/at15.0-0-alpha.ubuntu-20_x86_64_ppc64le/sources/gcc/libgcc/dfp-bit.c:32:10: fatal error: stdio.h: No such file or directory
   32 | #include <stdio.h>
      |          ^~~~~~~~~
compilation terminated.
```

 The culprit was this commit from GCC upstream:
```
commit 781183595acba67a37c66f59a0c1d9b5fee7e248
Author: Michael Meissner <meissner@linux.ibm.com>
Date:   Mon Feb 22 15:33:29 2021 -0500                                                                                                                                                                                                                       

Add conversions between _Float128 and Decimal.
```

The GCC developers are already aware of this, and a patch sets have already been posted to gcc-patches. A v3 is waiting for review ([patch1](https://gcc.gnu.org/pipermail/gcc-patches/2021-March/566312.html) and [patch2](https://gcc.gnu.org/pipermail/gcc-patches/2021-March/566314.html)).

**Comment by mscastanho** [comment]
[I just pinged](https://gcc.gnu.org/pipermail/gcc-patches/2021-March/567110.html) the gcc patch to see if gets more attention.

**Comment by mscastanho** [comment]
One patch has been committed. Then I tried a cross build with revision `645bfc161987` and the patch from [1]. The build succeeded finally!

Once that patch gets in, this problem should be fixed.

[1] https://gcc.gnu.org/pipermail/gcc-patches/2021-March/567453.html

**Comment by mscastanho** [comment]
This was fixed by #1990
